src/cpu/cpu_inner_product_list.cpp

```diff
@@ -27,6 +27,10 @@
 using namespace dnnl::impl::cpu::x64;
 #endif
 
+#if DNNL_AARCH64
+#include "cpu/aarch64/jit_brgemm_inner_product.hpp"
+using namespace dnnl::impl::cpu::aarch64;
+#endif
 #if DNNL_AARCH64 && DNNL_AARCH64_USE_ACL
 #include "cpu/aarch64/acl_inner_product.hpp"
 using namespace dnnl::impl::cpu::aarch64;
```

Comment:
(nitpicking) would  it make sense to remove the now redundant `using namespace dnnl::impl::cpu::aarch64;` that is under USE_ACL ?